build.gradle

```diff
@@ -82,8 +82,18 @@ allprojects {
       testRuntimeOnly "org.junit.jupiter:junit-jupiter-engine:" + junit_version
     }
 
-    tasks.withType(Test) {
+    tasks.withType(Test).configureEach {
       useJUnitPlatform()
+
```

Comment:
how do these confs help? can we ensure in the build scan that all `catalogTest` and `statementTest` are indeed run? 
We isolated these before because their sparksessions would interfere with each other.